test-clients/src/main/java/com/hedera/services/bdd/suites/contract/OZContracts/OZContract_ERC1155.java

```diff
@@ -0,0 +1,163 @@
+package com.hedera.services.bdd.suites.contract.OZContracts;
```

Comment:
I suggest we move this file to package com.hedera.services.bdd.suites.contract.openzeppelin and rename the file to ERC1155ContractInteractions to be consistent with other OZ related tests